packages/experimental/AppearanceAdditions/ios/FRNAppearanceAdditions.m

```diff
@@ -137,9 +166,20 @@ - (void)stopObserving {
 
 - (void)appearanceChanged:(NSNotification *)notification {
     if (_hasListeners) {
+        RCTBridge *notificationBridge = [[notification object] bridge];
+        if (![notificationBridge isKindOfClass:[RCTBridge class]]) {
+            return;
+        }
+
+        // Don't send the appearanceChanged event if the notification didn't originate from the same react native instance
+        RCTBridge *currentBridge = [[self bridge] parentBridge];
```

Comment:
where does parentBridge come from? I've never seen that before.

Reply:
[self bridge] actually returns a RCTxxBridge, which is only apparent at runtime. It's the C++ version of RCTBridge, and gets initialized with an RCTBridge (which is accessed with parentBridge)